.cloud-build/ExecuteChangedNotebooks.py

```diff
@@ -13,20 +13,112 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 import argparse
+import dataclasses
```

Comment:
GitHub won't let me put the comment on the line, but the first line `#!/bin/bash` is throwing off GitHub's syntax highlighting.

Should be `#!/usr/bin/env python`

Reply:
Good catch, let me fix those.